src/utils/strings.js

```diff
@@ -20,14 +20,19 @@
 
 import t from 'translations/translate';
 
+type FormattedValueOptions = {
+  isPositive?: boolean,
+  noSymbol?: boolean
+}
+
 export const capitalize = (str: string): string => {
   if (typeof str !== 'string') return '';
   return str.charAt(0).toUpperCase() + str.slice(1);
 };
 
-export const getValueWithSymbol =
-  (value: string, token: string, isPositive: boolean, doNotFormat: boolean) => {
-    if (doNotFormat) return t('tokenValue', { value, token });
-    if (isPositive) return t('positiveTokenValue', { value, token });
-    return t('negativeTokenValue', { value, token });
-  };
+export const getFormattedValue = (value: string, token: string, options: FormattedValueOptions = {}) => {
+  const { isPositive, noSymbol } = options;
+  if (noSymbol || isPositive === 'undefined') return t('tokenValue', { value, token });
```

Comment:
A missing `typeof`?

Reply:
Shoot, yup.
Fixing it and calling it a day :D
Thanks!